demoapp/www/js/crashes.js

```diff
@@ -119,15 +119,15 @@ $(document).bind('pageinit', function () {
         LowMemory.generateLowMemory();
     });
 
-    var updateAttachment = function () {
-        $('#text_attachment_value').html("Current value: " + text);
-        $('#file_attachment_value').html("Current value: " + attachment);
+    var updateAttachmentUI = function () {
+        $('#text_attachment_value').html("Current value: " + textAttachment);
```

Comment:
```suggestion
        $('#text_attachment_value').text("Current value: " + textAttachment);
```

If the textAttachment is html, you'll be rendering the actual html in the dom.